app/content_service/Ingestion/lectures_ingestion.py

```diff
@@ -0,0 +1,73 @@
+import base64
+from typing import Dict
+import fitz
+import weaviate
+from app.vector_database.lectureschema import init_lecture_schema, LectureSchema
+from ..Ingestion.abstract_ingestion import AbstractIngestion
+from app.llm import BasicRequestHandler
+
+
+class LectureIngestion(AbstractIngestion):  # Inherits from the abstract class
```

Comment:
This is not a useful comment
```suggestion
class LectureIngestion(AbstractIngestion):
```